subsys/bluetooth/host/Kconfig

```diff
@@ -95,7 +95,7 @@ config BT_HCI_RESERVE
 
 choice BT_RECV_CONTEXT
 	prompt "BT RX Thread Selection"
-	default BT_RECV_BLOCKING if BT_LL_SW_SPLIT || BT_H4
+	default BT_RECV_WORKQ_SYS if SOC_SERIES_NRF51X
```

Comment:
BBC Microbit did not have enough RAM even with the change to the controller's stack sizes. It compiles with `BT_RECV_WORKQ_SYS` in addition to the before-mentioned stack reduction, so it can use that.